app/views/scans/_form_additional.html.erb

```diff
@@ -1,5 +1,11 @@
-<%= f.text_field :page_range, help: 'Example: 1-5, 8, 7-13' %>
+<div class="form-group">
+  <%= f.label :page_range, class: 'control-label col-sm-3' %>
+  <div class="col-sm-9">
+    <%= f.text_field_without_bootstrap :page_range, class: 'form-control', 'aria-describedby' => 'page_range_help_block' %>
```

Comment:
The aria bit could use the new hash syntax by using `aria: { describedby: 'page_range_help_block' }`